share/doc/homebrew/Tips-N'-Tricks.md

```diff
@@ -136,3 +136,9 @@ inline patches in Vim.
 [homebrew-mode](https://github.com/dunn/homebrew-mode) provides syntax
 highlighting for inline patches as well as a number of helper functions
 for editing formula files.
+
+### Atom
+
+[language-hombrew-formula](https://atom.io/packages/language-hombrew-formula)
```

Comment:
Thanks! can you fix the package name here too and then we should be good to merge.